include/sys/arc.h

```diff
@@ -18,10 +18,13 @@
  *
  * CDDL HEADER END
  */
+ 
 /*
  * Copyright (c) 2005, 2010, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2012, 2016 by Delphix. All rights reserved.
- * Copyright (c) 2013 by Saso Kiselkov. All rights reserved.
+ * Copyright (c) 2012, 2016, Delphix. All rights reserved.
+ * Copyright (c) 2013, by Saso Kiselkov. All rights reserved.
+ * Copyright (c) 2019 by Klara Inc. All rights reserved.
```

Comment:
are we allowed/required to add that? I only cherry-picked Allans commits where no copyright was included